.github/workflows/helm-ci.yml

```diff
@@ -14,4 +14,3 @@ jobs:
       ct_configfile: operations/pyroscope/helm/ct.yaml
       ct_check_version_increment: false
       helm_version: v3.14.3
-      kubeVersion: "1.23"
```

Comment:
I wonder why this is a problem, I added it with this PR: https://github.com/grafana/pyroscope/pull/3159 to support the horizontal pod auto-scaler mode.

It is also not aligning with the version set here: https://github.com/grafana/pyroscope/blob/main/operations/pyroscope/helm/ct.yaml#L4

Reply:
https://github.com/grafana/pyroscope/actions/runs/9778263318

> [Invalid workflow file: .github/workflows/helm-ci.yml#L17](https://github.com/grafana/pyroscope/actions/runs/9778263318/workflow)
The workflow is not valid. .github/workflows/helm-ci.yml (Line: 17, Col: 20): Invalid input, kubeVersion is not defined in the referenced workflow.

`kubeVersion` is not a valid field, see https://github.com/grafana/helm-charts/blob/32ad8e1a78724cf1595090857b30d6032880d47a/.github/workflows/lint-test.yaml#L5-L28